### PR TITLE
Added code for cluster registration

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
PA-196
**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:
http://aetos.pwx.purestorage.com/resultSet/testSetID/8365
http://aetos.pwx.purestorage.com/resultSet/testSetID/8369
http://aetos.pwx.purestorage.com/resultSet/testSetID/8646

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -2857,7 +2857,7 @@ func CreateSourceAndDestClusters(cloudCred, orgID string) {
 		expect(err).NotTo(haveOccurred(),
 			fmt.Sprintf("Failed to get kubeconfig path for destination cluster. Error: [%v]", err))
 		logrus.Debugf("Save cluster %s kubeconfig to %s", destinationClusterName, dstClusterConfigPath)
-		CreateCluster(destinationClusterName, cloudCred, dstClusterConfigPath, orgID)
+		CreateCluster(destinationClusterName, dstClusterConfigPath, orgID, cloud_name, uid)
```
Comment (lsrinivas-pure) [inline tests/common.go:2860]: After create cluster are you doing any verifications that cluster is created successfully

Comment (snigdha-px) [reply tests/common.go:2860]: Added the verification

File: tests/backup/backup_test.go [hunk 2]
```diff
@@ -1854,8 +1862,8 @@ func CreateProviderClusterObject(provider string, kubeconfigList []string, cloud
 		kubeconfigPath, err := getProviderClusterConfigPath(provider, kubeconfigList)
 		Expect(err).NotTo(HaveOccurred(),
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:1863]: change this to dash.verify

File: tests/common.go [hunk 3]
```diff
@@ -2848,7 +2848,7 @@ func CreateSourceAndDestClusters(cloudCred, orgID string) {
 			fmt.Sprintf("Failed to get kubeconfig path for source cluster. Error: [%v]", err))
 
 		logrus.Debugf("Save cluster %s kubeconfig to %s", sourceClusterName, srcClusterConfigPath)
```
Comment (lsrinivas-pure) [inline tests/common.go:2850]: change it to log

File: tests/common.go [hunk 4]
```diff
@@ -2857,7 +2857,7 @@ func CreateSourceAndDestClusters(cloudCred, orgID string) {
 		expect(err).NotTo(haveOccurred(),
 			fmt.Sprintf("Failed to get kubeconfig path for destination cluster. Error: [%v]", err))
 		logrus.Debugf("Save cluster %s kubeconfig to %s", destinationClusterName, dstClusterConfigPath)
```
Comment (lsrinivas-pure) [inline tests/common.go:2859]: same here

**Review by lsrinivas-pure — APPROVED** [review]
minor comments plz fix and merge

File: tests/backup/backup_test.go (hunk 2 above)
Comment (snigdha-px) [reply tests/backup/backup_test.go:1863]: done

File: tests/common.go (hunk 3 above)
Comment (snigdha-px) [reply tests/common.go]: done

File: tests/common.go (hunk 4 above)
Comment (snigdha-px) [reply tests/common.go]: done

**Pull request merged**